packages/backend-defaults/src/entrypoints/auth/DefaultAuthService.ts

```diff
@@ -166,6 +168,9 @@ export class DefaultAuthService implements AuthService {
           });
         }
         // If the target plugin does not support the new auth service, fall back to using old token format
+        this.logger.warn(
+          'tokenManager is DEPRECATED, please migrate to the new auth service, see https://backstage.io/docs/tutorials/auth-service-migration for more information',
+        );
```

Comment:
```suggestion
        this.logger.warn(
          `DEPRECATION WARNING: A call to the '${targetPluginId}' plugin had to fall back to using deprecated auth via the token manager service. Please migrate all plugins to the new auth service, see https://backstage.io/docs/tutorials/auth-service-migration for more information`,
        );
```

Thiunking we can provide a bit more information here.